core/trace/agent.go

```diff
@@ -43,6 +45,11 @@ func StartAgent(c Config) {
 	agents[c.Endpoint] = lang.Placeholder
 }
 
+// StopAgent shuts down the span processors in the order they were registered.
+func StopAgent() {
+	_ = tp.Shutdown(context.TODO())
```

Comment:
Use Background()

Reply:
Uh-huh,Background() is better as the top-level Context.